src/guides/v2.4/release-notes/commerce-2-4-1.md

```diff
@@ -817,14 +959,94 @@ We have fixed hundreds of issues in the Magento 2.4.1 core code.
+
+<!-- ENGCOM-7663 -->
+
+*  You can use the `subscribeEmailToNewsletter` mutation to subscribe customers to a newsletter. See [`subscribeEmailToNewsletter` mutation](https://devdocs.magento.com/guides/v2.4/graphql/mutations/subscribe-email-to-newsletter.html). _Fix submitted by Alexander Taranovsky in pull request [27586](https://github.com/magento/magento2/pull/27586)_. [GitHub-27337](https://github.com/magento/magento2/issues/27337)
+
+<!-- ENGCOM-7216 -->
+
+*  Removed redundant logic in the `setShippingMethodsOnCart` mutation resolver. _Fix submitted by Alexander Taranovsky in pull request [27349](https://github.com/magento/magento2/pull/27349)_. [GitHub-28262](https://github.com/magento/magento2/issues/28262)
+
+<!-- ENGCOM-7638 -->
+
+*  Added test coverage for the `Please provide Email of sender` error associated with the `sendEmailToFriend` mutation. _Fix submitted by Alexander Taranovsky in pull request [28034](https://github.com/magento/magento2/pull/28034)_. [GitHub-28138](https://github.com/magento/magento2/issues/28138)
+
+<!-- ENGCOM-7743 34485-->
+
+*  The `products` query now returns product attribute option labels for the default store view as expected. Previously, this query returned the product attribute option labels that were set for the Admin. _Fix submitted by Dmitriy Gallyamov in pull request [28647](https://github.com/magento/magento2/pull/28647)_. [GitHub-28568](https://github.com/magento/magento2/issues/28568)
+
+<!-- ENGCOM-7743 -->
+
+*  Custom attribute aggregations now return store-specific option values. _Fix submitted by Dmitriy Gallyamov in pull request [28647](https://github.com/magento/magento2/pull/28647)_. [GitHub-28572](https://github.com/magento/magento2/issues/28572)
+
+<!-- ENGCOM-7707 -->
+
+*  Price intervals that are returned in a product aggregation can now be numeric only (no wildcards permitted). _Fix submitted by Dmitriy Gallyamov in pull request [28745](https://github.com/magento/magento2/pull/28745)_. [GitHub-28628](https://github.com/magento/magento2/issues/28628)
+
+<!-- ENGCOM-7729 -->
+
+*  The `categoryList` query now returns the correct response when using fragments. _Fix submitted by Ulzii in pull request [28710](https://github.com/magento/magento2/pull/28710)_. [GitHub-28584](https://github.com/magento/magento2/issues/28584)
+
+<!--- MC-34187-->
+
+*  The new `availableStores` query returns a list of `StoreConfigs` for other stores available under the same website (based on the current store). It does not expose the list of websites. See [`availableStores` query](https://devdocs.magento.com/guides/v2.4/graphql/queries/available-stores.html). [GitHub-28569](https://github.com/magento/magento2/issues/28569)
+
+<!-- ENGCOM-7810 MC-34187-->
+
```

Comment:
This is a duplicate of the previous item

Reply:
deleted